### PR TITLE
Python websockets

### PR DESCRIPTION
**Pull request by LetMyPeopleCode**
Adding a Quickstart Guide for using websockets with Python.

### CODE REVIEW

File: site/pmguides/src/websockets_python/websockets_python.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+The [WebSocket protocol](https://www.rfc-editor.org/rfc/rfc6455) defines how this is done. The connection is both persistent and low-latency, so communication can be initiated by the client or the server, instead of the client request / server response communication model many developers associate with the web.
+
+In this tutorial, you'll create a simple WebSocket server in Python, and use Postman to send and receive messages across the WebSocket connection.
+
+
+### Prerequisites
+
+- [Python 3](https://www.python.org/downloads/) installed on your machine
+
+### What You Will Learn
```
Comment (mjgutermuth) [inline site/pmguides/src/websockets_python/websockets_python.md:29]: Apologies for the nitpick, but for consistency, we use the contracted form for these headings ("What You'll Learn", "What You'll Need", etc)

File: site/pmguides/src/websockets_python/websockets_python.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+<!-- ------------------------ -->
+
+## Next Steps
+
+Duration: 1
+
+More and more developers are exploring [WebSocket APIs in Postman](https://learning.postman.com/docs/sending-requests/websocket/websocket/). According to Postman’s [State of the API 2022](https://www.postman.com/state-of-api/api-technologies/#api-technologies) report, WebSockets are used by 26% of respondents.
+
+![WebSockets are used by 26% of survey respondents](./assets/graph.png)
```
Comment (mjgutermuth) [inline site/pmguides/src/websockets_python/websockets_python.md:170]: This is great and I'm glad you included it! I'm not sure "Next Steps" is really an accurate heading here, though.

**Review by mjgutermuth — APPROVED** [review]
This looks great!

File: site/pmguides/src/websockets_python/websockets_python.md (hunk 1 above)
Comment (LetMyPeopleCode) [reply site/pmguides/src/websockets_python/websockets_python.md:29]: For some weird reason, "you'll" was making **one** section render the list below the heading as plain text with no bullets or indents. No matter what I did, it only worked if I removed the contraction. I updated all those headings to remove the contraction for consistency.

File: site/pmguides/src/websockets_python/websockets_python.md (hunk 2 above)
Comment (LetMyPeopleCode) [reply site/pmguides/src/websockets_python/websockets_python.md:170]: That's straight from the Node version. I kept it for consistency.

**Pull request merged**